websites/S/Spyke/metadata.json

```
@@ -0,0 +1,20 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.9",
+	"author": {
+		"id": "726466786916171796",
+		"name": "iamhv"
+	},
+	"service": "Spyke",
+	"description": {
+		"en": "Spyke - Community Discussion"
+	},
+	"url": "spyke.social",
```

Site supports `www` subdomain.
```suggestion
	"url": [
		"spyke.social",
		"www.spyke.social"
	],
```